src/PostgREST/ApiRequest.hs

```diff
@@ -132,6 +136,14 @@ userApiRequest schema req reqBody
       , iCookies = fromMaybe [] $ parseCookiesText <$> lookupHeader "Cookie"
       }
  where
+  (filters, rpcQParams) =
+    case action of
+      ActionInvoke{isReadOnly=True} -> (filter (liftM2 (||) (isEmbedPath . fst) (hasOperator . snd)) flts,
```

Comment:
That's an awesome function, it perfectly suits my case since I had a `a or b` and the negation of that is `~(a or b) = ~a and ~b`.